examples/routing_service/routing_service_file_adapter/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+#
+# (c) 2019 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+#  RTI grants Licensee a license to use, modify, compile, and create derivative
+#  works of the Software.  Licensee has the right to distribute object form
+#  only for use with RTI products.  The Software is provided "as is", with no
+#  warranty of any type, including any warranty for fitness for any purpose.
+#  RTI is under no obligation to maintain or support the Software.  RTI shall
+#  not be liable for any incidental or consequential damages arising out of the
+#  use or inability to use the software.
+#
+cmake_minimum_required(VERSION 3.11)
+set (CMAKE_CXX_STANDARD 11)
+project (FileAdapter)
+
+
+# Find RTI Connext dependencies
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CONNEXTDDS_DIR}/resource/cmake")
+find_package(
+    RTIConnextDDS "6.0.0"
+    REQUIRED
+    COMPONENTS
+        core
+        routing_service
+)
+
+# Define the library that will provide the adapter plugin
+add_library(
+    fileadapter
+    # The kind of library created depends on value of BUILD_SHARED_LIBS=ON|OFF
```

Comment:
I think this comment is not needed

Reply:
I know this is verbose and redundant. But if the customer isn't well versed with CMake, its better to say more.

Reply:
I think it is not a good idea because, if we follow that philosophy we will need to comment each single line.

Reply:
Ok.

Reply:
Pralhad I see your intent of trying to have a good example to guide users but it's true we can't comment on all the cmake basic concepts. What you can do though, potentially, is to add some Note in the readme indicating that this example is prepared to build a dynamic library but that static linking is also supported if you link RS statically into your application.

Reply:
Yes. There is already a mention of that in the README.md. This one was superfluous.